src/styles/_catalog.scss

```diff
@@ -1,22 +1,31 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  $cardWidth: 200px;
```

Comment:
This variable can be initialized outside the selector.

Reply:
variable initialized outside of selector